var/spack/repos/builtin/packages/py-tensorboard/package.py

```diff
@@ -16,13 +16,18 @@ class PyTensorboard(Package):
 
     maintainers = ['aweits']
 
+    version('2.7.0', sha256='5632812bb9450e5741083b5b7826244ffdb732fb5bce970d526c81874a3e7fb2')
+    version('2.6.0', sha256='3d1e0a05828b25c1c28bd90c73d981a0a65c6a5550510bc7983d03ab915e6503')
+    version('2.5.0', sha256='58c9e0c31062821ab1c02845c3b7902da92574ef7192d701b1828dacbe4ee610')
     version('2.4.1', sha256='736dc204aa292d221f5871077e60994a9a9ea8e33b841f0d754d510fe6cc7635')
     version('2.4.0', sha256='28a30794c1c797357b2086477394b59afa0b18ca48592ca3c0627f7f10536373')
     version('2.3.0', sha256='947a58702c2841eb4559637dbf8639633f79de9a0f422be9737f3563a1725440')
     version('2.2.0', sha256='d0dfbf0e4b3b5ebbc3fafa6d281d4b9aa5478eac6bac3330652ab6674278ab77')
 
-    depends_on('python@2.7:2.8,3.2:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.2:', type=('build', 'run'), when='@:2.5')
+    depends_on('python@3.6:', type=('build', 'run'), when='@2.6:')
     depends_on('bazel@2.1.0:', type='build', when='@2.2.0:')
+    depends_on('bazel@3.7.0:', type='build', when='@2.5.0:')
     depends_on('py-pip', type='build')
     depends_on('py-wheel', type='build')
```

Comment:
0.26+, at least in the latest version

Reply:
Ah, I removed the wrong line when I resolved the rebase conflicts.